02Lesson/01Homework/index.html

```diff
@@ -1,8 +1,54 @@
+            <h6> Narcos </h6>
```

Comment:
avoid adding unnecessary space in your text elements - in this case write it as `<h6>Narcos</h6>`. We use CSS to add spacing on either side of the text